.github/workflows/test.yml

```diff
@@ -0,0 +1,90 @@
+name: Tests
+
+on:
+  pull_request:
+  push:
+    branches: [master, main]
+
+jobs:
+  main:
+    runs-on: ubuntu-20.04
+    strategy:
+      # keep running so we can see if tests with other versions pass
+      fail-fast: false
+      matrix:
+        include:
+            # This version of k3s uses kubernetes v1.18
+            - k3s-version: v1.18.10+k3s2
+              helm_version: v3.4.0
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          # chartpress requires the full history
+          fetch-depth: 0
+      - uses: manics/action-k3s-helm@v0.2.1
+        with:
+          k3s-version: ${{ matrix.k3s-version }}
+          helm-version: ${{ matrix.helm-version }}
+          metrics-enabled: false
+          traefik-enabled: false
+          docker-enabled: true
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.8'
+      - name: Display kubernetes related details
+        shell: bash
+        run: |
+            echo "kubeconfig: $KUBECONFIG"
+            echo
+            echo "List of running pods:"
+            kubectl get pods --all-namespaces
+            echo
+            echo "List of cluster nodes:"
+            kubectl get nodes
+            echo
+            echo "Helm list output:"
+            helm list
+      - name: Setup helm chart repositories
+        shell: bash
+        run: |
+            helm repo add jupyterhub https://jupyterhub.github.io/helm-chart/
+            helm repo update
```

Comment:
This step is included in the k3s-helm action already actually

Reply:
Well, how can it know which repo to add and update it?

Reply:
Woops my mistake, I got thinking it installed the pebble-helm-chart as part of the github action, but it doesn't. Sorry for the noise.